SubZero/src/util/filter/HSVFilter.h

```diff
@@ -0,0 +1,112 @@
+/**
+  * HSVFilter.h
+  * Created on: June 19, 2015
+  * Author: James534
+  */
+
+#ifndef HSVFILTER_H
+#define HSVFILTER_H
+
+#include "Filter.h"
+#include <opencv2/imgproc.hpp>
```

Comment:
so... question, are we including opencv2? I've been using include in the opencv folder